Lesson19/gokce/script.js

```diff
@@ -0,0 +1,106 @@
+
+
+
+
+
+
+
+class ShoppingCart {
+  #items;
+  constructor() {
+    this.#items = [];
+  }
+  viewCart() {
+    console.log("Viewing cart....");
+    if (this.#items.length === 0) {
+      console.log("The cart is empty.");
+      return;
+    }
+    for (const item of this.#items) {
+      //console yazdırırken direk template literal kullanıp içine item yazamıyorsun çünkü item bir nesne o yüzden özelliklerine tek tek erişmen lazım)
+      console.log(
+        `> ${item.name} | Miktar: ${item.quantity} | Fiyat: ${item.price} | ID: ${item.id}`,
+      );
```

Comment:
Just as an alternative: since item is already the full object in the loop, you can simply use `console.log(item)` to print the entire object without accessing each property manually.
```suggestion
      //console yazdırırken direk template literal kullanıp içine item yazamıyorsun çünkü item bir nesne o yüzden özelliklerine tek tek erişmen lazım)
      console.log(item);
```